CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ This version contains **breaking** changes, as bugsnag-unity has been updated to
 
 
 ### Bug fixes
+* Timestamps in log messages
```

Comment:
I'd give this a general description, for those without context. Maybe something like:

```suggestion
* Fix an issue where timestamps and other `:`-containing log message content was interpreted as the error class
```